src/bioregistry/data/bioregistry.json

```diff
@@ -39897,12 +39903,19 @@
       ],
       "twitter": "gly_gen"
     },
-    "homepage": "https://glygen.org/glycan/",
+    "homepage": "https://glygen.org/",
+    "keywords": [
+      "genomics",
+      "glycomics",
+      "glycoproteomics",
+      "life science",
+      "proteomics"
+    ],
     "mappings": {
       "fairsharing": "FAIRsharing.aI1J5W",
       "uniprot": "DB-0254"
     },
-    "name": "Computational and Informatics Resources for Glycoscience",
+    "name": "GlyGen: Computational and Informatics Resources for Glycoscience",
```

Comment:
in general, the Bioregistry discourages the titles for prefixes from being written this way. What does GlyGen stand for?